src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_onnx.py

```diff
@@ -53,6 +67,8 @@ def __call__(
         latents: Optional[np.ndarray] = None,
         output_type: Optional[str] = "pil",
         return_dict: bool = True,
+        callback: Optional[Callable[[int, np.ndarray, torch.FloatTensor], None]] = None,
```

Comment:
```suggestion
        callback: Optional[Callable[[int, int, np.ndarray], None]] = None,
```

`timestep` as mentioned in the Stable Diffusion pipeline.

Regarding the latents, in this case the pipeline works with `numpy` arrays internally, see https://github.com/huggingface/diffusers/pull/654/files.

However, the call to the `scheduler` would return a tensor. I think we should convert to `numpy` for consistency and send that back. Do you agree @anton-l ?

Reply:
Okay sure, will add this change as well.